src/components/Rating.vue

```diff
@@ -13,9 +13,9 @@
               :max="max"
               :count="facet.count"
         >
-          <template v-for="(n, key) in max">
-            <span v-if="n <= facet.value" :class="bem('star')" :key="key">&#9733;</span>
-            <span v-else :class="bem('star', 'empty')" :key="key">&#9734;</span>
+          <template v-for="n in max">
+            <span v-if="n <= facet.value" :class="bem('star')" :key="n">&#9733;</span>
+            <span v-else :class="bem('star', 'empty')" :key="n">&#9734;</span>
```

Comment:
I don't know, I need to better understand how `key` is being used internally in Vue.
Because in the case of stars for example, the counts will evolve for a given key.

In any case, I feel like Vue is still able to see if the content of a key has changed an replaces it.